nibabies/interfaces/volume.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""This module provides interfaces for workbench volume commands"""
+import os
+
+from nipype.interfaces.base import TraitedSpec, File, traits, CommandLineInputSpec
+from nipype.interfaces.workbench.base import WBCommand
+from nipype import logging
+
+iflogger = logging.getLogger("nipype.interface")
+
+
+class CreateSignedDistanceVolumeInputSpec(CommandLineInputSpec):
+    surface = File(
+        exists=True,
+        mandatory=True,
+        argstr="%s ",
```

Comment:
I don't think you need to add a trailing space for these
```suggestion
        argstr="%s",
```